### PR TITLE
texlive: 2020.20210408 -> 2020.20210324 (tlnet-final)

### PR DESCRIPTION
**Pull request by veprbl**
###### Motivation for this change

Provide tlnet-final snapshot for use in 21.05

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dotlambda** [comment]
As this is a downgrade, wouldn't it make sense to wait until 21.05 is branched off?

**Comment by veprbl** [comment]
I am supposing that snapshot before the 2021 DVD release rush would be more "stable" for the time being.

**Comment by veprbl** [comment]
@GrahamcOfBorg builx tests.texlive

**Comment by veprbl** [comment]
@GrahamcOfBorg build tests.texlive

File: pkgs/tools/typesetting/tex/texlive/default.nix [hunk 1]
```diff
@@ -112,11 +112,12 @@ let
       # (https://tug.org/historic/).
       urlPrefixes = args.urlPrefixes or [
         # tlnet-final snapshot
-        #"http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2019/tlnet-final/archive"
-        #"ftp://tug.org/texlive/historic/2019/tlnet-final/archive"
+        "http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2019/tlnet-final/archive"
```
Comment (xworld21) [inline pkgs/tools/typesetting/tex/texlive/default.nix:115]:
```suggestion
        "http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/${bin.texliveYear}/tlnet-final/archive"
```
It should be 2020, right?

Comment (veprbl) [reply pkgs/tools/typesetting/tex/texlive/default.nix:115]: Yes, and it is.

**Comment by veprbl** [comment]
I guess, we are shipping 2021

**Pull request closed without merging**